blues/tests/test_mc.py

```diff
@@ -0,0 +1,128 @@
+                #                                 [0, 0, 1]])
+
+                #multiply lig coordinates by rot matrix and add back COM translation from origin
+                rot_move = np.dot(reduced_pos, set_rotation_matrix) * positions.unit + self.center_of_mass
+
+                # Update ligand positions in nc_sim
+                for index, atomidx in enumerate(self.atom_indices):
+                    positions[atomidx] = rot_move[index]
+                context.setPositions(positions)
+                positions = context.getState(getPositions=True).getPositions(asNumpy=True)
+                self.positions = positions[self.atom_indices]
+                return context
+
+
+        self.model = SetRotationMove(structure, resname='ALA')
+        #self.model = RandomLigandRotationMove(structure, resname='ALA')
+
+        self.model.atom_indices = range(22)
+        self.model.topology = structure[self.model.atom_indices].topology
+        self.model.positions = structure[self.model.atom_indices].positions
+        self.model.calculateProperties()
+
+        self.mover = MoveEngine(self.model)
+        #Initialize the SimulationFactory object
+        sims = SimulationFactory(structure, self.mover, **self.opt)
+        #print(sims)
+        system = sims.generateSystem(structure, **self.opt)
+        simdict = sims.createSimulationSet()
+        alch_system = sims.generateAlchSystem(system, self.model.atom_indices)
+        self.nc_sim = sims.generateSimFromStruct(structure, alch_system, ncmc=True, **self.opt)
+        self.model.calculateProperties()
+        self.initial_positions = self.nc_sim.context.getState(getPositions=True).getPositions(asNumpy=True)
+        asim = Simulation(sims, self.mover, **self.opt)
+        #monkeypatch to get log_ncmc acceptance
+        def nacceptRejectMC(self):
+            """Function that chooses to accept or reject the proposed move.
+            """
+            md_state0 = self.current_state['md']['state0']
+            md_state1 = self.current_state['md']['state1']
+            log_ncmc = (md_state1['potential_energy'] - md_state0['potential_energy']) * (-1.0/self.nc_integrator.kT)
+            randnum =  math.log(np.random.random())
+
+            if log_ncmc > randnum:
+                self.accept += 1
+                print('NCMC MOVE ACCEPTED: log_ncmc {} > randnum {}'.format(log_ncmc, randnum) )
+                self.md_sim.context.setPositions(md_state1['positions'])
+            else:
+                self.reject += 1
+                print('NCMC MOVE REJECTED: log_ncmc {} < {}'.format(log_ncmc, randnum) )
+                self.md_sim.context.setPositions(md_state0['positions'])
+            self.log_ncmc = log_ncmc
+            self.md_sim.context.setVelocitiesToTemperature(self.temperature)
+        asim.acceptRejectMC = nacceptRejectMC
+        nacceptRejectMC.__get__(asim)
+        asim.acceptRejectMC = types.MethodType(nacceptRejectMC, asim)
+        asim.runMC()
```

Comment:
So I'm confused. Why do we have `runMC` in this test, but much of the code just prior to this line talks about NCMC? Did you just forget to update the code when copying it from somewhere else? e.g., there is a print statement that says that the NCMC move was accepted, etc.